Source/utils/GRParmParse.hpp

```diff
@@ -127,6 +129,65 @@ class GRParmParse : public ParmParse
              std::vector<data_t>(num_comp, default_value));
     }
 
+    static bool folder_exists(const std::string &path)
+    {
+        struct stat sb;
```

Comment:
I think the `struct` keyword at instantiation is C-style. In C++, it should be ok to omit it. This object should also be given a more descriptive name (I can see you copied it from the example in the documentation for `stat`).

Reply:
Without `struct` I get an error, but I gave it a better name.